{"source": "exploit-database", "filename": "platforms/linux/remote/40507.py", "cves": ["CVE-2013-2088"]}